Entra SSO permissions (#9388)
fishtown-analytics/docs.getdbt.com: website/docs/docs/platform/manage-access/set-up-sso-microsoft-entra-id.md

| --- | --- | --- | --- |
| Microsoft Graph | Delegated | `User.Read` | Yes |
| Microsoft Graph | Delegated | `GroupMember.Read.All` | Yes |
| Microsoft Graph | Delegated | `Directory.Read.All` | Optional — may be required if users are assigned to > 200 groups |

</SimpleTable>

The default scope only requires `User.Read` and `GroupMember.Read.All`. If you assign a user to more than 200 groups, you may need to grant additional permissions such as `Directory.Read.All`. 

:::info SSO before December 2025

> If you set up SSO before December 2025, your existing configuration may still require `Directory.Read.All` rather than the `GroupMember.Read.All` scope shown for new setups. Alternatively, delete and re-create your SSO configuration to use the new targeted scopes. Refer to [Configuring permissions](#configuring-permissions) for details.

Reviewer: sugg simplification and updating to 'request' rather than require:
```suggestion
If you set up SSO before December 2025, your existing configuration may request `Directory.Read.All` instead of `GroupMember.Read.All` for new setups. To use the updated scopes, delete and re-create your SSO [configuration](#configuring-permissions).
```

<Expandable alt_header="Receiving a 'AADSTS90094: Admin consent is required' error">

> If you set up SSO before December 2025, your existing configuration may still require `Directory.Read.All` rather than the `GroupMember.Read.All` scope shown for new setups. Alternatively, delete and re-create your SSO configuration to use the new targeted scopes. Refer to [Configuring permissions](#configuring-permissions) for details.

Reviewer: ```suggestion
If you set up SSO before December 2025, your existing configuration may request `Directory.Read.All` instead of `GroupMember.Read.All` for new setups. To use the updated scopes, delete and re-create your SSO [configuration](#configuring-permissions).
```
